frontend/src/components/OdhAppCard.tsx

```diff
@@ -67,20 +69,20 @@ const OdhAppCard: React.FC<OdhAppCardProps> = ({ odhApp }) => {
         target="_blank"
         rel="noopener noreferrer"
       >
-        {`Launch `}
+        Launch
         <ExternalLinkAltIcon />
       </a>
       <a className={quickStartClasses} href="#" onClick={onQuickStart}>
-        Quick start
+        {getQuickStartLabel(odhApp.spec.quickStart, qsContext)}
       </a>
     </CardFooter>
   );
 
   const supportedImageClasses = classNames('odh-card__supported-image', {
-    'm-hidden': odhApp.spec.support !== 'redhat',
+    'm-hidden': odhApp.spec.category !== 'Red Hat',
```

Comment:
review/optional: on line 116 looks like your logic confirms the value is "defined"/"truthy" before also running this `!== 'Red Hat'... is that a concern here too?

Reply:
No, this is for anything not `Red Hat`, third party is for anything that has a category but it is not `Red Hat`.

P.S. - this is being changed in the near future due to design changes